### ISSUE TITLE
Memory issue during testing phase with custom dataset for semantic segmentation

### ISSUE THREAD

**Issue by Yatoronto** [Pointcept/Pointcept#290]
I’m encountering a memory issue during the testing phase of a semantic segmentation task on a custom dataset. The training phase proceeds without any issues, but during testing, the memory usage continuously increases until it overflows, leading to an error: RuntimeError: DataLoader worker (pid xxxxx) is killed by signal: Killed.

I suspect that the aug_transform in the testing script might be the cause of this issue. I’ve tried to disable it by directly deleting it, but then the test errors out with AttributeError: 'ConfigDict' object has no attribute 'aug_transform', forcing me to add it back.

How can I properly disable the aug_transform without causing this error?
Alternatively, what might be the real cause of the memory overflow during testing?

Any guidance would be greatly appreciated.

**Comment by Yatoronto** [comment]
New Progress: 
I am using the script `semseg-pt-v3m1-0-base` . I have removed the data augmentation operations from the `aug_transform` in the testing part of this script, leaving only an empty list `[]`. As a result, there is no memory overflow during testing. However, the test results are very poor, with several categories having an IOU/accuracy of 0, whereas the training process is normal.

**Comment by Gofinge** [comment]
Left one for inference, don't make it empty.

**Comment by blueeaglex** [comment]
@Gofinge  I also stuck by this and I found that reducing aug_transform is the only way.

**Comment by Gofinge** [comment]
> @Gofinge I also stuck by this and I found that reducing aug_transform is the only way.

Yes, determine the length of aug_transform by your computing resouce.
